src/test/java/com/wikia/webdriver/testcases/communityheadertests/CommunityHeaderTests.java

```diff
@@ -51,8 +53,28 @@ public void testLoggedInWikiButtons() {
     communityHeader.clickWikiActivity();
     Assert.assertTrue(driver.getCurrentUrl().contains("Special:WikiActivity"));
 
-    CreateArticleModalComponentObject modal = communityHeader.clickAddNewPage();
-    Assert.assertTrue(modal.isCreateNewArticleModalVisible());
+    CreateArticleModalComponentObject addNewPageModal = communityHeader.clickAddNewPage();
+    Assert.assertTrue(addNewPageModal.isCreateNewArticleModalVisible());
+    addNewPageModal.close();
+
+    communityHeader.openMoreToolsDropdown()
+        .clickMoreAddImageLink();
+    Assert.assertTrue(driver.getCurrentUrl().contains("Special:Upload"));
```

Comment:
these tests now might take a while to run, as it have to open all pages,
what if instead we'd check only if href is correct?

Reply:
@hakubo href may be correct but it does not mean that clicking on it will redirect you to destination page if some wired css-es or js-es are applied by accident